python/ray/tests/test_failure.py

```diff
@@ -799,3 +800,173 @@ def test(self):
 
     with pytest.raises(ray.exceptions.ObjectStoreFullError):
         ray.put(np.zeros(10**8 + 2, dtype=np.uint8))
+
+
+@pytest.mark.parametrize(
+    "ray_start_cluster", [{
+        "object_store_memory": 150 * 1024 * 1024,
+        "num_nodes": 2,
+        "num_cpus": 5,
+        **generate_internal_config_map(
+            object_manager_repeated_push_delay_ms=1000,
+            initial_reconstruction_timeout_milliseconds=200,
+        )
+    }, {
+        "object_store_memory": 150 * 1024 * 1024,
+        "num_nodes": 1,
+        "num_cpus": 10,
+    }],
+    indirect=True)
+def test_direct_call_eviction(ray_start_cluster):
+    object_store_memory = 150 * 1024 * 1024
+    num_objects = 10
+
+    @ray.remote
+    def create_object(size):
+        return np.random.rand(size)
+
+    # Submit enough tasks so that they exceed the size of the object store.
+    objects = []
+    for _ in range(num_objects):
+        obj = create_object.remote(object_store_memory // num_objects)
```

Comment:
Aren't these missing a .options(is_direct_call=True), or is the idea they all run with RAY_FORCE_DIRECT=1?

Reply:
Oh oops, yeah I was running it with RAY_FORCE_DIRECT=1 locally, but I meant to change it to use the options.